src/index.js

```diff
@@ -1,12 +1,16 @@
+/* eslint-disable no-unused-vars */
 import EmailClientController from './core/email-client-controller';
-import SiftView from './core/sift-view.js';
-import SiftController from './core/sift-controller.js';
-import SiftStorage from './core/sift-storage.js';
+import SiftView from './core/sift-view';
+import SiftController from './core/sift-controller';
+import SiftStorage from './core/sift-storage';
+import { useSiftController, useSiftView } from './react';
 
 export { EmailClientController };
 export { SiftController };
 export { SiftStorage };
 export { SiftView };
+export { useSiftController };
+export { useSiftView };
 
 /**
  * SiftView
```

Comment:
I don't understand the purpose of `registerSiftView` and `registerSiftController`, they just log to console... but I've left them in the usage docs for the new functions just in case...